src/test/groovy/net/ravendb/pages/QueryDataExplorationPage.groovy

```diff
@@ -0,0 +1,78 @@
+package net.ravendb.pages
+
+import geb.Page
+
+
+class QueryDataExplorationPage extends Page {
+
+    final static String QUERY_COLLECTION_OPTION_CATEGORIES = "Categories"
+    final static String QUERY_RESULTS_COLUMN_ID = "Id"
+    final static String QUERY_RESULTS_COLUMN_NAME = "Name"
+    final static String QUERY_RESULTS_COLUMN_DESCRIPTION = "Description"
+
+    static at = {
+        runButton
+    }
+
+    static content = {
+        // tool bar
+        runButton { $("button[title='Run']") }
+        selectCollectionButton { $("button[title='Collection']") }
+        collectionDropdown { $("ul.dropdown-menu[data-bind='foreach: collections']") }
+        dropdownOptionSelector { "a" }
```

Comment:
this is used only once in the method below - please remove this property and use "a" directly in the method's code